config/routes.rb

```diff
@@ -1,10 +1,14 @@
 Rails.application.routes.draw do
   root "static_pages#home"
-  get "/help", to: "static_pages#help"
-  get "/home", to: "static_pages#home"
-  get "/about", to: "static_pages#about"
-  get "/contact", to: "static_pages#contact"
-  get "/signup", to: "users#new"
-  post "/signup", to: "users#create"
-  resources :users, only: [:create, :show]
+  get "help", to: "static_pages#help"
+  get "home", to: "static_pages#home"
+  get "about", to: "static_pages#about"
+  get "contact", to: "static_pages#contact"
+  get "signup", to: "users#new"
+  post "signup", to: "users#create"
+  get "users", to: "users#new"
```

Comment:
dòng này không cần thiết nữa, vì dòng 7:  `get "signup", to: "users#new"` nó làm nhiệm vụ đó rồi ạ